test-agents/copilotstudio-webchat/src/index.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import path from 'path'
+
+import { loadCopilotStudioConnectionSettingsFromEnv } from '@microsoft/agents-copilotstudio-client'
+import pkg from '@microsoft/agents-copilotstudio-client/package.json' with { type: 'json' }
+import express from 'express'
+
+import { dependency } from './util.js'
+
+const PORT = process.env.PORT || 3978
+const settings = loadCopilotStudioConnectionSettingsFromEnv()
+const settingsString = JSON.stringify(settings, null, 2)
+
+const app = express()
+
+// Serves all static assets from the "public" directory to support the website UI.
+app.use(express.static(path.resolve(import.meta.dirname, '../public')))
+
+// Make the Copilot Studio Client browser library available from node_modules.
```

Comment:
With ES6 support in the browser, and importmaps, we dont need to load anything from node_modules, and we could avoid this custom middleware